4.extract_features/pipelines/OP_pipe.cppipe

```diff
@@ -59,22 +59,35 @@ Groups:[module_num:4|svn_version:'Unknown'|variable_revision_number:2|show_windo
     grouping metadata count:1
     Metadata category:Plate
 
-ConvertImageToObjects:[module_num:5|svn_version:'Unknown'|variable_revision_number:1|show_window:False|notes:['This module converts the imported binary mask images into objects to extract information from.']|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
+ConvertImageToObjects:[module_num:5|svn_version:'Unknown'|variable_revision_number:1|show_window:False|notes:[]|batch_state:array([], dtype=uint8)|enabled:True|wants_pause:False]
     Select the input image:Mask
     Name the output object:ConvertImageToObjects
```

Comment:
Recommend changing the name here to something that represents the actual objects in the masks, maybe `bones`?